src/components/ColorPicker/components/ColorPickerContent/ColorPickerContentComponent.jsx

```diff
@@ -51,19 +53,24 @@ const ColorPickerContentComponent = forwardRef(
     }, [isBlackListMode, colorsList]);
 
     const onColorClicked = useCallback(
-      color => {
+      (color, removeOnAdd) => {
```

Comment:
this doesn't feel right  - usually, an "onClick" callback only receives the item which was clicked. Maybe we need to rename this function? (or perhaps the variable?)